src/ast/Content.ts

```diff
@@ -8,4 +8,12 @@ import {AstNode} from "./AstNode";
  *
  * e.g. DIR | CLASS| INTERFACE
  */
-export abstract class Content extends AstNode {}
+export abstract class Content extends AstNode {
+    // Path to this directory, class or interface file
+    parentPath: string;
```

Comment:
I think ModuleDecl could use this in evaluate, so maybe parentPath should be pulled up to AstNode

Reply:
Why does ModuleDec need this?